llm/prompt/template/openai/chat.py

```diff
@@ -38,3 +39,65 @@ def get_stylish_completion(prompt, style="Passionate and friendly"):
     messages = prompt_template.format_messages(style=style, prompt=prompt)
     response = chat(messages)
     return response.content
+
+
+def llmChain(prompt_input, chain_input):
+    llm = ChatOpenAI(temperature=0.9, model=llm_model)
+    prompt = ChatPromptTemplate.from_template(prompt_input)
+    chain = LLMChain(llm=llm, prompt=prompt)
+
+    return chain.run(chain_input)
+
+
+def simpleSequentialChain(prompt1, prompt2, chain_input):
+    llm = ChatOpenAI(temperature=0.9, model=llm_model)
+
+    first_prompt = ChatPromptTemplate.from_template(prompt1)
+    chain_one = LLMChain(llm=llm, prompt=first_prompt)
+
+    second_prompt = ChatPromptTemplate.from_template(prompt2)
+    chain_two = LLMChain(llm=llm, prompt=second_prompt)
+
+    overall_simple_chain = SimpleSequentialChain(
+        chains=[chain_one, chain_two], verbose=True
+    )
+
+    return overall_simple_chain.run(chain_input)
+
+
+def sequentialChain(prompt1, prompt2, prompt3, prompt4, review):
+    llm = ChatOpenAI(temperature=0.9, model=llm_model)
+
+    first_prompt = ChatPromptTemplate.from_template(prompt1)
+    # chain 1: input= Review and output= English_Review
+    chain_one = LLMChain(
+        llm=llm, prompt=first_prompt, output_key="English_Review"
+    )
+
+    second_prompt = ChatPromptTemplate.from_template(prompt2)
+    # chain 2: input= English_Review and output= summary
+    chain_two = LLMChain(
+        llm=llm, prompt=second_prompt, output_key="summary"
+    )
+
+    third_prompt = ChatPromptTemplate.from_template(prompt3)
+    # chain 3: input= Review and output= language
+    chain_three = LLMChain(
+        llm=llm, prompt=third_prompt, output_key="language"
+    )
+
+    fourth_prompt = ChatPromptTemplate.from_template(prompt4)
+    # chain 4: input= summary, language and output= followup_message
+    chain_four = LLMChain(
+        llm=llm, prompt=fourth_prompt, output_key="followup_message"
+    )
+
+    # and output= English_Review,summary, followup_message
+    overall_chain = SequentialChain(
+        chains=[chain_one, chain_two, chain_three, chain_four],
+        input_variables=["Review"],
+        output_variables=["English_Review", "summary","followup_message"],
```

Comment:
I notice the response example with some keys showing capitals, seems like they are following by the keys provide here. We should consider align keys with all lowercase letters

Reply:
okay